dags/hivemind_discord_summary_research.py

```
@@ -0,0 +1,210 @@
+            f"Processing (research) community_id: {community_id} | platform_id: {platform_id} | collection: {collection_name}"
+        )
+
+        # Cleanup collections if requested
+        if cleanup_collections:
+            logging.info(
+                f"Cleaning up (research) collections for community_id: {community_id} | platform_id: {platform_id}"
+            )
+            # Use the base cleanup and rely on research suffix for isolation in Qdrant naming
+            try:
+                cleanup_discord_summary_collections(community_id, platform_id)
+            except Exception as cleanup_error:  # pragma: no cover - defensive
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)